Gemfile

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+source "https://rubygems.org"
+
+git_source(:github) {|repo_name| "https://github.com/#{repo_name}" }
+
+# Manage our dependency on the version of the github-pages gem here.
+gem "github-pages", "= 222"
+
+# Explicitly include this gem here.
+# It is not directly included in the github-pages gem list of dependencies,
+# even though it is included in the original GitHub Pages build infrastructure.
+gem "jekyll-include-cache", "= 0.2.1"
```

Comment:
nit: does it make sense to add this to the `github-pages` gem deps instead?

Reply:
This one had some [history](url). I think it could be, but I also did it this way to show how we could include the things we needed independently of the pages-gem if needed. I'm ok going either way.